gum/backend-darwin/gumprocess-darwin.c

```diff
@@ -1885,6 +1885,7 @@ gum_emit_import (const GumImportDetails * details,
 {
   GumEnumerateImportsContext * ctx = user_data;
   GumImportDetails d;
+  Dl_info info;
```

Comment:
(You should add yourself to the copyright header in this file by the way.)

Reply:
oh yeah right, because the string we use doesn't actually live on the stack